packages/devtools_app/lib/src/shared/routing.dart

```diff
@@ -185,6 +215,7 @@ class DevToolsRouterDelegate extends RouterDelegate<DevToolsRouteConfiguration>
     routes
       ..clear()
       ..add(configuration);
+    notifyListeners();
```

Comment:
this will have side effects in `setNewRoutePath` where `_replaceStack` was also called but without calling `notifyListeners` after.

Reply:
Thought this was necessary, but just double checked and it seems like it isn't. Reverted the `notifyListeners` changes.